BioForm/api/permissions.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.permissions import BasePermission
+
+
+class IsJWTAuthenticated(BasePermission):
+    message = 'You must be logged in.'
+
+    def has_permission(self, request, view):
+        serializer = request.api_user
+        if serializer:
+            return True
+        return False
+
+class IsUsersObject(IsJWTAuthenticated):
+
+    def has_object_permission(self, request, view, obj):
+        serializer = request.api_user
+        return obj.user.id == serializer.data['id']
```

Comment:
By doing `obj.user.id` it will make a database request to get the user record to avoid it you simply can do `obj.user_id` and it will be directly obtained from obj. As obj is a generic object it may not have `user_id` attribute and it will break the code to make it safe you can use [`getattr`](https://www.programiz.com/python-programming/methods/built-in/getattr).
```suggestion
        return getattr(obj, 'user_id') == serializer.data['id']
```

Reply:
Changed as per suggestion. getattr also used to get user's id from request.api_user as well